SU-SWS/saa_victoria_subtheme
# ADAPT-000 Fixed banner and removed .idea/ folder

# READY FOR REVIEW

# Summary
- Remove .idea/*
- Add `.idea/*` to .gitignore
- Add `.saa-banner--no-card` class to image-only styling

# Review By (Date)
- 06/18

# Urgency
- medium

# Steps to Test

1. Pull this branch to your local
2. Check homepage and confirm that it appears as expected (see [Banner C: left](https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=192%3A627))

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
